src/types/graph.types.ts

```diff
@@ -1,4 +1,5 @@
 export type GraphConfig = {
     x_field: string,
     y_field: { aggragation?: "valuesCount" | "uniqueValues"; field?: string; };
+    dataFields?: { field: string, label: string; }[];
```

Comment:
this is the item type rite? if you think it might be used anywhere again create the type here too

Reply:
actually I didnt use the label so I remove